android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -78,6 +79,19 @@
   private boolean mSecondScanComplete;
   private boolean mFirstTime;
   private DiscoveryServiceConnection mDiscoveryServiceConnection;
+  private Comparator<PwPair> distanceSort = new Comparator<PwPair>(){
+                                              @Override
```

Comment:
move this waaay over...I think only two spaces is needed here for an indent, when creating a class like this.